### PR TITLE
OCPBUGS-17088: Baremetal IPI install doc is missing required port access information

### PR DESCRIPTION
**Pull request by johnwilkins**
Added a section for required open ports.

Fixes: OCPBUGS-17088 and [TELCODOCS-1389](https://issues.redhat.com//browse/TELCODOCS-1389)

See https://issues.redhat.com/browse/OCPBUGS-17088  and https://issues.redhat.com/browse/TELCODOCS-1389 for additional details.

Preview URL: http://184.23.213.161:8080/OCPBUGS-17088/installing/installing_bare_metal_ipi/ipi-install-prerequisites.html#network-requirements-ensuring-required-ports-are-open_ipi-install-prerequisites

For release(s): 4.14, 4.13
QE Review: 

- [x] QE has approved this change. 

Signed-off-by: John Wilkins <jowilkin@redhat.com>

### CODE REVIEW

File: modules/ipi-install-network-requirements.adoc
Comment (elfosardo) [inline modules/ipi-install-network-requirements.adoc:25]: 6180 and 6183 give access to the same backend, it's just the former is not secured, the latter is

File: modules/ipi-install-network-requirements.adoc
Comment (elfosardo) [inline modules/ipi-install-network-requirements.adoc:29]: I would mention 6388 as well with 6385

**Review by elfosardo — CHANGES_REQUESTED** [review]

File: modules/ipi-install-network-requirements.adoc
Comment (elfosardo) [inline modules/ipi-install-network-requirements.adoc:23]: 5051 is a private port, let's not mention it

File: modules/ipi-install-network-requirements.adoc
Comment (elfosardo) [inline modules/ipi-install-network-requirements.adoc:25]: let's say that 6183 is the preferred option and this is needed to get the inspection and deployment images

File: modules/ipi-install-network-requirements.adoc
Comment (elfosardo) [inline modules/ipi-install-network-requirements.adoc:27]: 6388 is a private, let's not mention it
6385 is used as either secured or not, the secure option is the preferred one

**Review by elfosardo — CHANGES_REQUESTED** [review]
sorry John, had to review again as I made a mistake on the exposed ports
thanks!

**Comment by carbonin** [comment]
@elfosardo do we also want to mention 9999 here?

**Comment by elfosardo** [comment]
> @elfosardo do we also want to mention 9999 here?

yes, let's mention it here

port 9999 is needed to reach the ironic python agent from the metal3 pod, specifically from the ironic service

**Comment by johnwilkins** [comment]
@elfosardo Can you have another look. I've conflated this with TELCODOCS-1389. There are quite a few more ports we needed to document.

File: modules/ipi-install-network-requirements.adoc
Comment (kcarmichael08) [inline modules/ipi-install-network-requirements.adoc:16]:
```suggestion
Certain ports must be open between cluster nodes for installer-provisioned installations to complete successfully. In certain situations, such as using separate subnets for far edge worker nodes, you must ensure that the nodes in these subnets can communicate with nodes in other subnets on the following required ports.
```

File: modules/ipi-install-network-requirements.adoc
Comment (kcarmichael08) [inline modules/ipi-install-network-requirements.adoc:37]:
```suggestion
|`6385`| The Ironic API server runs initially on the bootstrap VM and later on the control plane nodes and listens on port `6385`. The Ironic API allows clients to interact with Ironic for bare metal host provisioning and management, including operations like enrolling new hosts and managing their power state, deploying images, and cleaning the hardware.
```

File: modules/ipi-install-network-requirements.adoc
Comment (kcarmichael08) [inline modules/ipi-install-network-requirements.adoc:25]:
```suggestion
| `69` | When using a provisioning network, cluster nodes communicate with the TFTP server on port `69` using their provisioning network interfaces. The TFTP server runs on the bootstrap VM. The bootstrap VM runs on the provisioner node.
```

File: modules/ipi-install-network-requirements.adoc
Comment (kcarmichael08) [inline modules/ipi-install-network-requirements.adoc:197]:
```suggestion
The out-of-band management IP address is on a separate network from the node. To ensure that the out-of-band management IP address can communicate with the provisioner node during installation, the out-of-band management IP address must be granted access to port `6180` on the provisioner node and on the {product-title} control plane nodes. TLS port `6183` is required for virtual media installation, for example, by using Redfish.
```

Comment (kcarmichael08) [reply modules/ipi-install-network-requirements.adoc:197]: Per IBM Style, "via" should only be used to refer to routing data. https://www.ibm.com/docs/en/ibm-style?topic=word-usage#v

**Review by kcarmichael08 — COMMENTED** [review]
I marked a couple of typos and a style issue. Please squash into one commit before requesting merge review. Thanks!

**Comment by kcarmichael08** [comment]
I also see that it was reviewed so please check the QE approved box if that is the case.

**Review by kelbrown20 — COMMENTED** [review]
Updates LGTM!

**Pull request merged**
